### PR TITLE
Update piteas-tokenlist.json

### PR DESCRIPTION
**Pull request by KolomboPulse**
add please

### CODE REVIEW

**Comment by piteasio** [comment]
Low liquidity/volume/rate for listing. Rejected!

File: piteas-tokenlist.json [hunk 1]
```diff
@@ -1798,6 +1798,14 @@
             "symbol": "UFO",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x456548A9B56eFBbD89Ca0309edd17a9E20b04018.png"
+        },
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1801]: },

File: piteas-tokenlist.json [hunk 2]
```diff
@@ -1798,6 +1798,14 @@
...
+        {
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1802]: {

File: piteas-tokenlist.json [hunk 3]
```diff
@@ -1798,6 +1798,14 @@
             "symbol": "UFO",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x456548A9B56eFBbD89Ca0309edd17a9E20b04018.png"
+        },
+        {
+            "chainId": 369,
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1803]: "chainId" : 369 ,

File: piteas-tokenlist.json [hunk 4]
```diff
@@ -1798,6 +1798,14 @@
             "symbol": "UFO",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x456548A9B56eFBbD89Ca0309edd17a9E20b04018.png"
+        },
+        {
+            "chainId": 369,
+            "name": "6 Tons Of Digital Gold",
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1804]: "name": "6 Tons Of Digital Gold",

File: piteas-tokenlist.json [hunk 5]
```diff
@@ -1798,6 +1798,14 @@
             "symbol": "UFO",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x456548A9B56eFBbD89Ca0309edd17a9E20b04018.png"
+        },
+        {
+            "chainId": 369,
+            "name": "6 Tons Of Digital Gold",
+            "address": "0x4FC0C8a7636cfE551bb077BA928b4E9d59b02DC2",
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1805]: "address": "0x4FC0C8a7636cfE551bb077BA928b4E9d59b02DC2",

File: piteas-tokenlist.json [hunk 6]
```diff
@@ -1798,6 +1798,14 @@
             "symbol": "UFO",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x456548A9B56eFBbD89Ca0309edd17a9E20b04018.png"
+        },
+        {
+            "chainId": 369,
+            "name": "6 Tons Of Digital Gold",
+            "address": "0x4FC0C8a7636cfE551bb077BA928b4E9d59b02DC2",
+            "symbol": "GOLD6T",
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1806]: "symbol": "GOLD 6T",

File: piteas-tokenlist.json [hunk 7]
```diff
@@ -1798,6 +1798,14 @@
             "symbol": "UFO",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x456548A9B56eFBbD89Ca0309edd17a9E20b04018.png"
+        },
+        {
+            "chainId": 369,
+            "name": "6 Tons Of Digital Gold",
+            "address": "0x4FC0C8a7636cfE551bb077BA928b4E9d59b02DC2",
+            "symbol": "GOLD6T",
+            "decimals": 8,
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1807]: "decimals": 8,

File: piteas-tokenlist.json [hunk 8]
```diff
@@ -1798,6 +1798,14 @@
...
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x456548A9B56eFBbD89Ca0309edd17a9E20b04018.png"
+        },
+        {
+            "chainId": 369,
+            "name": "6 Tons Of Digital Gold",
+            "address": "0x4FC0C8a7636cfE551bb077BA928b4E9d59b02DC2",
+            "symbol": "GOLD6T",
+            "decimals": 8,
+            "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x4FC0C8a7636cfE551bb077BA928b4E9d59b02DC2.png"
```
Comment (KolomboPulse) [inline piteas-tokenlist.json:1808]: "logoURI": "https://raw.githubusercontent.com/piteasio/app-tokens/main/token-logo/0x4FC0C8a7636cfE551bb077BA928b4E9d59b02DC2.png"

**Review by KolomboPulse — COMMENTED** [review]
Update piteas-tokenlist.json

**Pull request closed without merging**